lib/presentation/screens/tasks_screen.dart

```diff
@@ -1,49 +1,169 @@
+// import 'package:flutter_todo/infrasturcture/local_database/drift_database.dart';
+// import 'package:flutter_todo/infrasturcture/task/task_repo_impl.dart';
+// import 'package:flutter_todo/presentation/screens/add_task_screen.dart';
+// import 'package:flutter_todo/presentation/widgets/tasks_list.dart';
+
+
+// class TasksScreen extends StatefulWidget {
+//   //  final TaskRepoImpl taskRepo;
+//   TasksScreen({Key? key}) : super(key: key );
+
+//   @override
+//   State<TasksScreen> createState() => _TasksScreenState();
+// }
+
+// class _TasksScreenState extends State<TasksScreen> {
+// // void _addTask(BuildContext context){
+// //   showModalBottomSheet(context: context, builder: (context)=>SingleChildScrollView(
+// //     child: Container(padding: EdgeInsets.only(
+// //       bottom: MediaQuery.of(context).viewInsets.bottom
+// //     ),
+// //     child: AddTaskScreen(),),
+// //   ));
+// // }
+
+//   @override
+//   Widget build(BuildContext context) {
+//     return Scaffold(
+//       body:  Scaffold(
+//           appBar: AppBar(
+//             title: Text("Tasks App"),
+//           ),
+//           body: Column(
+//             crossAxisAlignment: CrossAxisAlignment.center,
+//             children: [
+              
+//             ],
+//           ),
+
+//           // floatingActionButton: FloatingActionButton(
+            
+//           //   onPressed: (value){
+//           //             final updatedTask = task.copyWith(isCompleted: value);
+//           //             taskRepo.save(task: updatedTask);
+                   
+//           //   },
+//           //   tooltip: 'Add Task',
+//           //   child: Icon(Icons.add),
+//           // ),
+//         )
+//     );
+//     // BlocBuilder<>(
+//     //   builder: (context, state) {
+        
+        
+//     //   },
+//     // );
+//   }
+// }
+
 import 'package:flutter/material.dart';
-import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_todo/domain/task/entities/task.dart';
+import 'package:flutter_todo/infrasturcture/task/task_repo_impl.dart';
 import 'package:flutter_todo/presentation/screens/add_task_screen.dart';
-import 'package:flutter_todo/presentation/widgets/tasks_list.dart';
 
+class TaskPage extends StatefulWidget {
+  final TaskRepoImpl taskRepo;
 
-class TasksScreen extends StatefulWidget {
-  TasksScreen({Key? key}) : super(key: key);
+  TaskPage({required this.taskRepo});
 
   @override
-  State<TasksScreen> createState() => _TasksScreenState();
+  _TaskPageState createState() => _TaskPageState();
 }
 
-class _TasksScreenState extends State<TasksScreen> {
-void _addTask(BuildContext context){
-  showModalBottomSheet(context: context, builder: (context)=>SingleChildScrollView(
-    child: Container(padding: EdgeInsets.only(
-      bottom: MediaQuery.of(context).viewInsets.bottom
-    ),
-    child: AddTaskScreen(),),
-  ));
-}
+class _TaskPageState extends State<TaskPage> {
+  late Stream<List<Task>> taskStream;
```

Comment:
remove all logic or implement it in application layer and explain me why is wrong do it in presentation